dist/angular-patternfly.js

```diff
@@ -3205,6 +3205,26 @@ angular.module('patternfly.sort').directive('pfSimpleSort', function () {
     },
     templateUrl: 'sort/simple-sort.html',
     controller: ["$scope", function ($scope) {
+      var setSortTooltip = function () {
+        var tooltip = '';
+
+        if ($scope.config.currentField.sortType === 'numeric') {
+          if ($scope.config.isAscending) {
+            tooltip = 'Sort numerically, from least to greatest';
+          } else {
+            tooltip = 'Sort numerically, from greatest to least';
+          }
+        } else {
+          if ($scope.config.isAscending) {
+            tooltip = 'Sort alphabetically, from A to Z';
+          } else {
+            tooltip = 'Sort alphabetically, from Z to A';
+          }
+        }
+
+        $scope.sortTooltip = tooltip;
```

Comment:
What about putting these in the HTML?  Seems odd to have this sort of stuff in a controller.